app/components/memorable_date_component.rb

```diff
@@ -122,6 +122,14 @@ def i18n_long_format(date)
     end
   end
 
+  def has_errors?
+    form.object.respond_to?(:errors) && form.object.errors.key?(name)
+  end
+
+  def error_msg
+    form.object.errors[name]&.join('')
```

Comment:
Can you describe how this `join` affects the error message output?

Reply:
@NavaTim , the object itself is a single element array.

Reply:
I know that it's likely a single-element array in this case, but I'd like to confirm the behavior in the general case as well, e.g. if there are multiple backend errors for the same field. This component is coded in a way that facilitates reusability for other screens in the application.

Reply:
@NavaTim , this is one to consider, most cases it will be a single element. If not it's joined as one piece for now since this is the only one use case? 

Option1. on UI rendering, render the message(s) as needed, like a ul?
Option2. combine potential multiple entries into one, this is what's done here.

We can argue, when it ends in backend validation, the validation part already knows what's the requirement and create a single message.

Reply:
Given that the component uses behavior ultimately based on `ValidatedFieldElement`, it may be good to use only the first message in the array similar to how [`getNormalizedValidationMessage`](https://github.com/18F/identity-idp/blob/a92bc0b3c39698e90452ddeb7d08e29590215c15/app/javascript/packages/validated-field/validated-field-element.ts#L117) only uses the first matching error message. This would also reduce the likelihood of error messages being combined in an unintended manner.